dali/test/python/test_dali_tf_dataset_mnist_graph.py

```diff
@@ -29,7 +29,7 @@ def test_keras_single_cpu():
     run_keras_single_device('cpu', 0)
 
 
-@raises(Exception)
+@raises(Exception, "TF device and DALI device mismatch. TF*: CPU, DALI*: GPU for output*")
```

Comment:
You don't need to add trailing `*` at the end of glob pattern.

Reply:
done